src/AccountTrackerController.ts

```diff
@@ -3,7 +3,6 @@ import PreferencesController from './PreferencesController';
 import { BNToHex, safelyExecute } from './util';
 
 const EthjsQuery = require('ethjs-query');
-const BlockTracker = require('eth-block-tracker');
```

Comment:
I think we can finally remove it from the dependencies

Reply:
you're right! removed eth-block-tracker and `test.js` that seems like is not doing nothing.